src/utp_stream.cpp

```diff
@@ -44,9 +44,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/cstdint.hpp>
 #include <limits>
 
-#define TORRENT_UTP_LOG 0
-#define TORRENT_VERBOSE_UTP_LOG 0
```

Comment:
At this line https://github.com/arvidn/libtorrent/blob/master/src/utp_stream.cpp#L94

UTP_LOG and UTP_LOGV are defined and used in the rest of the file.